charter.html

```diff
@@ -178,24 +203,55 @@ <h3 id=chair-selection>Chair Selection</h3>
 <a href=https://www.w3.org/People/functions/w3m#dom>Dominique
 Hazaël-Massieux</a>.
 
-<h2 id=deliverables>Deliverables</h2>
+<h2 id=proposals>Proposals</h2>
+
+<p>A <dfn>Proposal</dfn> is an idea brought to the Community Group for
+consideration and potential adoption as a <a href=#work-items>Work
+Item</a>.
+
+<p>Any Community Group Participant may make a Proposal by filing
+<a href=https://github.com/privacycg/proposals/issues>an issue</a> in
+the
+<a href=https://github.com/privacycg/proposals><code>privacycg/proposals</code></a>
+repository on GitHub, stating the problem they would like to address and
+how they propose to address it.
 
-<p>The <a href=#chairs>Chairs</a> may add Deliverables, but must not add
-Deliverables which lack the support of at least
-two <a href=#implementer>implementers</a>.
+<p>The Community Group may discuss the Proposal on GitHub and during
+teleconferences or face-to-face meetings.
 
-<p>The Chairs may remove Deliverables.
+<p>The <a href=#chairs>Chairs</a> may create a dedicated repository for
+a Proposal. They may do this for any reason, such as but not limited to:
 
-<p>This document will be updated to reflect the current set of
-Deliverables of the Community Group.
+<ul>
+<li>because the Proposal has become sufficiently complex that writing an
```

Comment:
I imagine that a proposal author might just want to proactively create an explainer, and would naturally drop it into the proposals repo--so no need to get their own repo just to do that. Having a formal explainer, even in the proposal repo, allows the proposer an opportunity to refine and adapt the proposal if it is (for example) rejected by the chairs, or not enticing to enough implementers.

Reply:
They could even simply use the explainer as the text of the issue. I figure that's how a bunch of explainers will happen—the issue gets edited over time to reflect the discussion, and it ends up more and more explainer-y.

Reply:
Yep, that's what I figured you had in mind. I was just thinking that if a proposer wants to allow PRs and better change-tracking, they would want to make a separate code file--they're not limited to 1) tracking proposals as issues or 2) waiting to request a repo from the chairs.

Reply:
Sure, we could possibly do that too, I suppose. Do you think we need to call that out in the charter?